src/time_integration/methods_2N.jl

```diff
@@ -158,6 +158,9 @@ function solve!(integrator::SimpleIntegrator2N)
     end
   end
 
+  return TimeIntegratorSolution((first(prob.tspan), integrator.t),
+                                (copy(prob.u0), copy(integrator.u)),
```

Comment:
Do you really need a `copy` here or shall we just get the pointers without allocating additional memory (i.e. remote `copy` here)?

Reply:
Thanks. Done in 7462369a.